<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Static Microphone Volume</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/qt_resource_container.qrc">
    <normaloff>:/images/images/microphone-black-shape.ico</normaloff>:/images/images/microphone-black-shape.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #1e1e1e;         /* Dark solid background */
    color: #f0f0f0;                    /* Default light text color */
}

QWidget:hover {
    /* Optional: slight brightness change on hover if needed */
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QComboBox" name="devicesList">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    background-color: #2b2b2b;         /* Dark background */
    color: #f0f0f0;                    /* Light text color */
    border: 1px solid #444444;         /* Subtle border */
    border-radius: 6px;                 /* Rounded corners */
	padding-left: 5px;
}

QComboBox:hover {
    border: 1px solid #5a9bd5;         /* Blue border on hover */
    background-color: #303030;          /* Slightly lighter background */
}

QComboBox:focus {
    border: 3px solid #5a9bd5;         /* Blue border on focus */
    background-color: #303030;          /* Slightly lighter background */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;                        /* Width of the drop-down button */
    border-left: 1px solid #444444;     /* Divider between text and arrow */
    border-radius: 0 6px 6px 0;         /* Match combo box rounding */
    background-color: #2b2b2b;          /* Match background */
}

QComboBox::down-arrow {
    image: url(:/images/images/down-arrow.png); /* Arrow image, white */
    width: 12px;
    height: 12px;
}</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>...</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLineEdit" name="volumeValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Audiowide</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #2b2b2b;      /* Dark background */
    color: #f0f0f0;                  /* Light text color */
    border: 1px solid #444444;       /* Subtle border */
    border-radius: 6px;              /* Rounded corners */
    padding: 5px 8px;                /* Inner padding */
    selection-background-color: #5555aa; /* Text selection color */
    selection-color: white;          /* Text selection text color */
}

QLineEdit:focus {
    border: 3px solid #5a9bd5;       /* Blue border on focus */
    background-color: #303030;       /* Slightly lighter on focus */
}
</string>
              </property>
              <property name="placeholderText">
               <string>From 0% to 100%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbVolume">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Audiowide</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Volume</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLineEdit" name="intervalValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Audiowide</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #2b2b2b;      /* Dark background */
    color: #f0f0f0;                  /* Light text color */
    border: 1px solid #444444;       /* Subtle border */
    border-radius: 6px;              /* Rounded corners */
    padding: 5px 8px;                /* Inner padding */
    selection-background-color: #5555aa; /* Text selection color */
    selection-color: white;          /* Text selection text color */
}

QLineEdit:focus {
    border: 3px solid #5a9bd5;       /* Blue border on focus */
    background-color: #303030;       /* Slightly lighter on focus */
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="placeholderText">
               <string>In seconds</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbInterval">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Audiowide</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Interval</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbrun">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: #ffffff;
	border-radius: 22px;
	background: #212121;
	border: 1px solid #444444;       /* Subtle border */
}

QPushButton:hover {
	background-color: #2a2a2a;
}

QPushButton:pressed {
	color: #666666;
	background-color: #1a1a1a;
	border: 3px solid #5a9bd5;
	padding-top: 0.5em;
}</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbminimize">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: #ffffff;
	border-radius: 22px;
	background: #212121;
	border: 1px solid #444444;       /* Subtle border */
}

QPushButton:hover {
	background-color: #2a2a2a;
}

QPushButton:pressed {
	color: #666666;
	background-color: #1a1a1a;
	border: 3px solid #5a9bd5;
	padding-top: 0.5em;
}</string>
          </property>
          <property name="text">
           <string>Minimize</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources/qt_resource_container.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbminimize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
